tests/mocha/server/RoundTest.js

```diff
@@ -0,0 +1,52 @@
+MochaWeb.testOnly(function() {
+  describe('Round', function() {
+    var lunch = makeRound({ title: 'lunch' });
+
+    it('format()', function() {
+      chai.expect(makeRound({ formatCode: '2' }).format().name).to.equal("Best of 2");
+      chai.expect(makeRound({ formatCode: 'a' }).format().name).to.equal("Average of 5");
+      chai.expect(lunch.format()).to.be.undefined;
+    });
+
+    it('properties()', function() {
+      chai.expect(makeRound({ roundCode: '1' }).properties().name).to.equal("First round");
+      chai.expect(makeRound({ roundCode: 'g' }).properties().name).to.equal("Combined Third Round");
+      chai.expect(lunch.properties()).to.be.undefined;
+    });
+
+    it('eventName()', function() {
+      chai.expect(makeRound({ eventCode: '333bf' }).eventName()).to.equal("Rubik's Cube: Blindfolded");
+      chai.expect(lunch.eventName()).to.equal("");
+    });
+
+    it('eventSolveTimeFields()', function() {
+      chai.expect(makeRound({ eventCode: '333mbf' }).eventSolveTimeFields().join()).to.equal("millis,puzzlesSolvedCount,puzzlesAttemptedCount");
+      chai.expect(lunch.eventSolveTimeFields()).to.be.undefined;
+    });
+
+    it('status convenience functions', function() {
+      var unstarted = makeRound({ status: 'unstarted' });
+      var open      = makeRound({ status: 'open' });
+      var closed    = makeRound({ status: 'closed' });
```

Comment:
Aligning code is banned at Google. It's derisively called "ASCII art". To me, increased readability always wins over some extra writing effort, but I know that's a minority view and I'll change this to the project standard.